docs/Release-Guide.md

```diff
@@ -91,8 +91,7 @@ Bumping the limits may significantly improve build speed.
    and this release contains all the needed binaries and checksums.
 1. Update the release description with the content of [CHANGELOG](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/CHANGELOG.md) for this release.
 1. Publish release by pressing "Publish release" green button in GitHub's UI.
-1. Bump version of the VictoriaMetrics cluster in the [sandbox environment](https://github.com/VictoriaMetrics/ops/blob/main/gcp-test/sandbox/manifests/benchmark-vm/vmcluster.yaml)
```

Comment:
I'd keep this link even if it is unreachable for public users.
wdyt @zekker6 @f41gh7 ? Do you think it somehow exposes us from sec perspective?

Reply:
This hypothetically gives more information about our environment, but I don't see any issues with that. There is no way to get unauthorized access to this link anyway, so there is no harm in keeping it in public repo.

If there are concerns regarding keeping this link as a public one I would suggest adding a release guide doc in ops repo and add reference to this doc here.

Reply:
@hagen1778 @zekker6 what do you think about moving it from docs, that a published to a website to READMEs which are only in a repo?

Reply:
Makes sense to me. Do we reference this links somewhere else? Can it be removed safely?